admin_interface/templates/django-admin-autocomplete-filter/autocomplete-filter.html

```diff
@@ -0,0 +1,11 @@
+{#
+ copied from https://github.com/farhan0581/django-admin-autocomplete-filter/ to highlight active
```

Comment:
Can’t copy this, the project is under GPL!

Reply:
You can rewrite the same logic without copying it directly.

Reply:
To override the template I need to copy and change it, so I won’t be taking any risk for your project and handle the override in my site.